package.json

```diff
@@ -80,6 +80,7 @@
     "download": "^5.0.2",
     "filesize": "^3.3.0",
     "fs-extra": "^0.26.7",
+    "get-stdin": "^5.0.1",
```

Comment:
yeah, not sure if it's doable with `readline` but for sure with `process.stdin`. I've decided to do that because it's small module, without dependencies.